doc/references/release-notes.rst

```diff
@@ -7,6 +7,8 @@ Upcoming Release
 
 .. warning:: The features listed below are not released yet, but will be part of the next release! To use the features already you have to install the ``master`` branch, e.g. ``pip install git+https://github.com/pypsa/pypsa#egg=pypsa``.
 
+* Introduce the functionality of parameters to the statistics module. With this one can set the parameters `nice_names`, `drop_zero` and `round` which is then applied to all statistics methods without the need to set them individually.
```

Comment:
Include a usage example in the release notes.

Reply:
I have added a usage example here